packages/dashboard/src/components/account-button-toggle/accounts-button-toggle.scss

```diff
@@ -0,0 +1,32 @@
+.button-toggle {
+    &__container {
+        width: 20.8rem;
+        height: 4.8rem;
+        margin: 3rem auto;
+        padding: 0.8rem;
+        background-color: var(--general-section-1);
+        border-radius: 0.8rem;
+    }
+    &__items {
+        .dc-button-menu {
+            &__wrapper {
+                background-color: unset !important;
+            }
+            &__button {
+                width: 9.6rem;
+                height: 3.2rem;
+                span {
+                    font-size: var(--text-size-s) !important;
+                    line-height: var(--text-lh-m) !important;
+                }
+                &--active span {
+                    color: var(--general-main-1);
+                }
+            }
+            &--highlight {
+                background-color: $color-apple-blue !important;
```

Comment:
Blue Apple? :D

Reply:
That's the name designers put as well :-)